python/dolfinx/mesh.py

```diff
@@ -333,6 +334,37 @@ def refine(
     return Mesh(mesh1, mesh._ufl_domain)
 
 
+def refine_interval(
+    mesh: Mesh,
+    cells: typing.Optional[np.ndarray] = None,
+    redistribute: bool = True,
+    ghost_mode: GhostMode = GhostMode.none,
```

Comment:
The default should be `none` - all our meshes are by default constructed with `shared_facet`.

Reply:
That's a typo? Will switch to `shared_facet`.

Reply:
Indeed!